src/drtsans/tof/eqsans/chopper.py

```
@@ -75,7 +69,9 @@ def __init__(self, other):
             ch = self._choppers[chopper_index]
             ch.offset = self._offsets[self.frame_mode][chopper_index]
 
-    def transmission_bands(self, cutoff_wl: float = None, delay: float = 0, pulsed: bool = False) -> Wbands:
+    def transmission_bands(
+        self, cutoff_wl: float = None, delay: float = 0, emission_delay: EmissionDelay = None
```

Ruff (0.15.15)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)